testing/examples/state_tests.rs

```diff
@@ -89,13 +89,23 @@ async fn main() {
 	let _keys = api.get_keys(storage_key, None).unwrap().unwrap();
 	let _constants: Balance = api.get_constant("Balances", "ExistentialDeposit").unwrap();
 
-	let max_keys = 3;
+	let max_keys = 2003;
+	let result =
+		api.get_storage_keys_paged(Some(storage_key_prefix.clone()), max_keys.clone(), None, None);
+	assert!(result.is_err());
```

Comment:
I don't think we need to check for an error here. That's an error from Substrate itself, which may change over time

Reply:
That's true, good input.